virt/virt-2-4-release-notes.adoc

```diff
@@ -18,6 +18,10 @@ include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 
 //CNV-5371 - Import existing virtual machines from Red Hat Virtualization to OpenShift Virtualization.
 
+
+//CNV-5438 - svirt-support-for-cnv
+Security has been significantly enhanced in this release. {VirtProductName} now supports SELinux with Mandatory Access Control (MAC) for isolating virtual machines (VMs). Previously, all VMs were managed by using privileged xref:../security/container_security/security-platform.adoc#security-deployment-sccs_security-platform[Security Context Constraints (SCC)]. Now, you can use less privileged custom SCCs for VMs and limit the use of privileged SCC to infrastructure containers in the cluster.
```

Comment:
For the first sentence, I would say something like this (active voice): "This release features significant security enhancements."

There should be a bullet point at the beginning of the note (I think I missed this in your other PR as well).

One other nit: inconsistent pluralization when using the SCC acronym. It looks like all three instances should be `SCCs`.